packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../devtools.dart' as devtools;
+import '../../flutter/common_widgets.dart';
+import '../../flutter/screen.dart';
+import '../../version.dart';
+import '../info_controller.dart';
+
+class InfoScreen extends Screen {
+  const InfoScreen() : super('Info');
+
+  @override
+  Widget build(BuildContext context) => InfoScreenBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      icon: Icon(Icons.info),
+      text: 'Info',
+    );
+  }
+
+  /// The key to identify the flag list view
+  @visibleForTesting
+  static const Key flagListKey = Key('Info Screen Flag List');
+
+  /// The key to identify the flutter version view.
+  @visibleForTesting
+  static const Key flutterVersionKey = Key('Info Screen Flutter Version');
```

Comment:
why include the word "Info Screen" in the key name?

Reply:
Just to be sure in a test we don't pick out a key that belongs to something else.